src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProvider.cs

```diff
@@ -120,11 +120,13 @@ public async override Task<OrchestrationStatusQueryResult> GetOrchestrationState
 
         public override bool ValidateDelayTime(TimeSpan timespan, out string errorMessage)
```

Comment:
This code path gets called for everything that involves creating a durable timer, right?

Reply:
Yes. The only other place this limitation was coded was in the default max retry interval in RetryOptions. Because that was a customer-facing value, and we don't have access to the durability provider at the time of creation, we had to choose a universal default for the constructor. That being said, this method is called on the various `TimeSpan` fields of the RetryOptions object when the object is passed into the `DurableOrchestrationContext` methods.